src/i-client-config.ts

```diff
@@ -104,10 +105,41 @@ export interface IPrecomputedClientConfig extends IBaseRequestConfig {
 }
 
 /**
- * Configuration for regular client initialization
- * @public
+ * Base options for the EppoClient SDK
  */
-export interface IClientConfig extends IBaseRequestConfig {
+export type IApiOptions = {
+  /**
+   * Your key for accessing Eppo through the Eppo SDK.
+   */
+  sdkKey: string;
```

Comment:
change from current `apiKey`. newly named `sdkKey` is used to differentiate configuration structs passed to the EppoJSClient constructor.
the type `IClientConfig` omits this field and uses `apiKey` instead to preserve backwards compatibility.